conda/recipes/cuml/meta.yaml

```diff
@@ -43,7 +43,7 @@ requirements:
     - libcumlprims {{ minor_version }}
     - cupy>=7.8.0,<9.0.0a0
     - treelite=1.0.0
-    - nccl>=2.5
+    - nccl>=2.8.4
```

Comment:
Can you please leave out unrelated changes from this pull request?

Reply:
sorry about that, don't know why i thought rebasing was a good idea!